helix-term/src/commands.rs

```diff
@@ -225,7 +225,7 @@ impl MappableCommand {
         find_prev_char, "Move to previous occurrence of char",
         extend_till_prev_char, "Extend till previous occurrence of char",
         extend_prev_char, "Extend to previous occurrence of char",
-        repeat_last_motion, "repeat last motion(extend_next_char, extend_till_char, find_next_char, find_till_char...)",
+        repeat_last_motion, "Repeat last motion",
```

Comment:
Somewhat unrelated to the change here, this tends to be a source of confusion that `A-.` doesn't work on motions like `w`, just on these find-type motions (same behavior as Kakoune). Maybe we should have a different word for those kinds of motions?